lib/p2p/Pool.ts

```diff
@@ -580,7 +583,16 @@ class Pool extends EventEmitter {
     peer.sendPacket(new packets.GetOrdersPacket({ pairIds: this.handshakeData.pairs }));
     if (this.config.discover) {
       // request peer's known nodes only if p2p.discover option is true
-      peer.sendPacket(new packets.GetNodesPacket());
+      if (this.config.discoverminutes === 0) {
+        // timer is disabled
+        peer.sendPacket(new packets.GetNodesPacket());
+        peer.getNodesTimer = undefined; // defensive programming
+      }else {
```

Comment:
should add white space between `}` and `else`

Reply:
I don't know why tslint didn't warn me, but done :)